sources/Makefile.am

```diff
@@ -13,9 +13,9 @@ sources =
 
 build_sources = AssemblyInfo.cs $(sources)
 
-CLEANFILES = $(DLL) generated-stamp generated/*.cs $(API)
+CLEANFILES = $(DLL) generated-stamp generated/* $(API)
 
-DISTCLEANFILES = AssemblyInfo.cs $(DLLMAP)
+DISTCLEANFILES = AssemblyInfo.cs
```

Comment:
I don't know yet, investigating. The patch author replied, but he couldn't explain the removal from `EXTRA_DIST`.